lib/models/BucketInfo.ts

```diff
@@ -37,6 +37,41 @@ export type VersioningConfiguration = {
     MfaDelete: any;
 };
 
+export type VeeamCapacity = {
```

Comment:
type should be either `VeamCapability` (to make it generic and support other veam capabilities in the feature) or `VeamSosApi` (and we would possibly have other VeamXXXX capabilities in the future)